### ISSUE TITLE
Hotkey or quick way to get to Github view of Pull Requests

### ISSUE THREAD

**Issue by bradical** [ZenHubIO/support#1216]
Really liking the new Zenhub webapp. (Nice work guys!)

Though one thing I noticed is that the Pull Request view is not particularly useful in Zenhub and I find myself usually immediately switching to the Github view. Could we get a hotkey/shortcut or a right-click or somthing that we could use to quickly get to the Github view of Pull Requests (or Issues though these are pretty usable in Zenhub).

Thanks for your consideration!

**Comment by ryanhiebert** [comment]
I came to look for approximately this as well. I'm using the deep browser GitHub integration, and it's beautiful, but when I want to look at a particular issue, it's showing me the issue in a modal instead of just navigating to that page on GitHub. Is there a way for me to do that? If I were able to set a user setting (perhaps with an organization default as well) to make the links to open the issue just take me to the GitHub page for that issue, that'd be what I'm looking for.

**Comment by bradical** [comment]
Should note that there _is_ a "View this issue on Github" link which works well but requires that i first open it in a modal. So yea, I think the key is a way to open a link to an issue/PR directly on Github.com from the board.

**Comment by ryanhiebert** [comment]
@bradical : Thanks, I now see that the title of the modal is a clickable link to the issue. That's definitely helpful. I agree that it would be nice to not have to open the modal first. HuBoard, for example, links the issue number specially in the cards to link them to the issue. I think that would work nicely.

**Comment by ryanhiebert** [comment]
For me, with the Chrome extension, it links to the GitHub issue. Sounds like it's different for you.

**Comment by bradical** [comment]
Sorry, I'm talking about the web version: app.zenhub.com

**Comment by ryanhiebert** [comment]
Yeah, that's fine, I suspect that the answer will be similar for both of our cases.

**Comment by kendramoroz** [comment]
Great discussion here and helpful insight on how our community is using app.zenhub.com! I'll take this feedback back to the team to see how we can improve the PR workflow and add updates here as they become available. Keep the feedback coming 👍 !
